styleguide/source/_patterns/01-molecules/09-search-results/_search-results.scss

```diff
@@ -0,0 +1,130 @@
+.search-result {
+  margin-bottom: $space;
+  width: 100%;
+  display: flex;
+  cursor: pointer;
+
+  &:active,
+  &:hover {
+    .search-result_link {
+      text-decoration: underline;
+    }
+
+    .search-result_left {
+      background-color: $black-15;
+    }
+  }
+}
+
+.search-result_label {
+  text-transform: uppercase;
+  color: $purple;
+  margin-bottom: 20px;
+
+  @include breakpoint($bp-small) {
+    margin-bottom: 25px;
+  }
+}
+
+.search-result_title {
+  @include type($font-serif, 18px, $font-weight-medium, 1.2222222222);
```

Comment:
Switching the title to use a h3 with the default type sizing which matches that sizing.